docs/assemblies/assembly_configuring-a-thorntail-application.adoc

```diff
@@ -23,6 +23,13 @@ include::../modules/proc_setting-system-properties-using-the-command-line.adoc[l
 
 include::../modules/proc_specifying-jdbc-drivers-for-hollow-jars.adoc[leveloffset=+2]
 
+[id='configuring-a-thorntail-application-using-environment-variables_{context}']
```

Comment:
But I just followed the pattern as just below this the YAML section creates an anchor ID as well.

Reply:
Those anchor shouldn't be there either, it's an oversight on my part when I was creating those files. Since there are some xrefs pointing to them, I'll address that in a separate issue. Feel free to leave the ID in as I'll deal with it then.